src/redux/actions/types.js

```diff
@@ -72,3 +72,4 @@ export const SET_PERVIOUS_PAGE = '[page] prev_page';
 export const SET_NEXT_PAGE = '[page] next_page';
 export const SET_CURRENT_PAGE = '[page] current_page';
 export const IS_LOADING = '[ui] loader_spinning';
+export const UN_POPULATE_THIS_ANSWER = '[concept] unpopulate an answer';
```

Comment:
What does unpopulate mean?

Reply:
Its an action type I just created. It sets the prePopulated flag to false so that the concept can be edited - after its has mapping has been removed.